### DISCUSSION TITLE
呜呜～

### DISCUSSION THREAD

**Discussion by liugunshi** [share121/inter-knot#799] [其他]
![mmexport1723259103197](https://github.com/user-attachments/assets/a896fd16-6da7-48ab-80f1-6d8d7e606819)
不会英语，发帖子好难

**Comment by capacitor1** [comment]
![image](https://github.com/user-attachments/assets/0168f8e7-b966-498f-847f-4caa6e850021)

当然绳网上面也有解决方法：[InterKnotForm@723 - 兄弟们，github可以改语言吗🤔，全是英文看着好难受啊](https://github.com/share121/inter-knot/discussions/723)

**Comment by themagicmoon** [comment]
可以去下个github的应用，那个有中文，就是没有绳网的界面而已，其他的都挺好的

**Comment by Fzy85** [comment]
呜，花费了几十分钟，不过以后就容易多了

**Comment by SDCOM-0415** [comment]
一旦用习惯了就好了
